benchmarks/random.h

```diff
@@ -0,0 +1,20 @@
+// Generating random data
+
+#pragma once
+
+#include <algorithm>
+#include <cstdint>
+#include <functional>
+#include <random>
+#include <vector>
+
+
+::std::vector<::std::uint64_t> GenerateRandom64(::std::size_t count) {
+  ::std::vector<::std::uint64_t> result(count);
+  ::std::random_device random;
+  auto genrand = [&random]() {
+    return random() + (static_cast<::std::uint64_t>(random()) << 32);
+  };
```

Comment:
It produces different results.

For the standard library implementation I have installed, default_random is a linear congruential generator; the comments call it "The classic Minimum Standard rand0 of Lewis, Goodman, and Miller." That cuts the false positive rates in half. I believe this is due to my use of the multiply-shift hash function. For a similar example, see Patrascu and Thorup's "The Power of Simple Tabulation Hashing", Figure 2: https://people.csail.mit.edu/mip/papers/charhash/charhash.pdf. It shows that multiply shift behaves better on structured input with some seeds, and much worse with other seeds.